affectedmoduledetector/src/main/kotlin/com/dropbox/affectedmoduledetector/AffectedModuleDetector.kt

```diff
@@ -113,8 +111,39 @@ abstract class AffectedModuleDetector {
         private const val ENABLE_ARG = "affected_module_detector.enable"
         var isConfigured = false
 
+        private var affectedModuleConfiguration: AffectedModuleConfiguration? = null
+
+        @JvmStatic
+        fun getRootConfiguration(project: Project): AffectedModuleConfiguration {
+            if (affectedModuleConfiguration == null) {
+                var rootProject = project
+                while (!rootProject.isRoot || rootProject.parent != null) {
+                    rootProject = rootProject.parent!!
+                }
```

Comment:
This `while` loop is unnecessary. `project` has a `getRootProject()` in it that is more efficient (rootProject is stored as a variable on the class so no need for a while loop